starfish/test/full_pipelines/recipe/iss_recipe.txt

```diff
@@ -0,0 +1,54 @@
+primary_image = file_inputs[0]
```

Comment:
This is great. We need to doc how this works and what your mini-language can do.

Reply:
#1245